### ISSUE TITLE
[BUG] Wrong error msg in other dimensions.

### ISSUE THREAD

**Issue by Chinthor** [ldtteam/minecolonies#8567]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.18

### MineColonies Version

minecolonies-1.18.2-1.0.919-ALPHA

### Structurize Version

structurize-1.18.2-1.0.424-ALPHA

### Related Mods and their Versions

1.18.2-forge-40.1.69
blockui-1.18.2-0.0.58-ALPHA
domum_ornamentum-1.18.2-1.0.55-ALPHA-universal
DireWolf20 Pack MC1.18 - Pack v1.9.1

### Current Behavior

Establish your first colony in any dimension. Overworld, Nether, End, Modded Mining Dimension, Modded Void Dimension, RFTools Custom Dimension. When trying to start a second colony in a different dimension, the GUI shows that I do not own a colony (yes I do) and you are able to create a colony. Clicking the [Create New Colony] button gives the error "Colony creation failed, try somewhere else!"
No amount of moving or repositioning the town hall corrects the error.
![image](https://user-images.githubusercontent.com/8145428/185242366-9355dc5f-759d-42e5-848f-73e9a2eaa841.png)
![image](https://user-images.githubusercontent.com/8145428/185242444-7a1a35c7-9a23-401e-9174-113d7078aed8.png)

### Expected Behavior

What should appear is the message and GUI explaining that you already have a colony and you need to abandon it before starting this new one, same as if your second colony was in the same dimension as the first. Ideally, this is all included in the first GUI window that (wrongly) said you are able to make a colony here.

### Reproduction Steps

Start a colony in any dimension.
Attempt to start a second colony in a different dimension.

### Logs

https://gist.github.com/Chinthor/e1c05873744fd7cf4c10e09fc8b033a4#file-gistfile1-txt

### Anything else?

As a related note, the option to abandon a colony in the GUI could maybe warn that all colonists will die, and refer to ways to avoid that, along with any warnings about bugginess and server lag. Like the setAbandoned command or the easy abandon option in the cfg

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by uecasm** [comment]
Each dimension is entirely separate.  You don't own a colony in that dimension yet, thus it correctly says you don't have any colony.

You should be able to start a colony in another dimension without issue, AFAIK.  Did you try placing a supply camp first?

**Comment by Chinthor** [comment]
True that I don't have a colony in this dimension, but that misses the point. It is the other colony in the other dimension that is preventing me from continuing. You might have been able to do one in every dimension in past versions, but not now. You must abandon your first before your alternate dimension colony can start. The bigger problem is the misleading error.

Supply camp or not has no effect.

**Comment by dragyomony** [comment]
Greetings,
I am getting the same error that your images show and i have not made any colonies before. I have tried in many ares with and without supply camps on flat and cleared out locations. i am playing on a server and do not have access to alot of functions so i am unsure where to go from here, or if this information helps at all.

**Comment by Raycoms** [comment]
then you have to turn on other dim colonies in the config
